### ISSUE TITLE
Less variables not being imported

### ISSUE THREAD

**Issue by jpants** [prncc/antd-scss-theme-plugin#63]
I've been working to get Antd running in Next.js with SCSS with cssModules.  I have been able to get it all running except for getting the Antd Less variables.  I can overwrite the Less variables from my 'theme.scss' file, but when I import that file, I can't get access to the Less variables as expected.  I get errors complaining of a Sass error because a variable doesn't exist.  

Any idea when this plugin will get some TLC? Lots of outdated deps...

**Comment by urrri** [comment]
I have the same problem.
sass-loader: 7.3.1
this plugin: 1.0.8

It also doesn't work with sass-loaded 8.0.0

**Comment by jpants** [comment]
yeah, I had to drop down sass-loader down to 7.3.1 as well.  The compile time with this plugin was too long, so I ended up using a different approach anyway.  Ended up being less sweet than what this plugin offers, but at least I'm not waiting a while after every little scss change
